.gitignore

```diff
@@ -1,6 +1,8 @@
 build/*
 lib/gmock-1.7.0/*
 lib/pbjson/*
+lib/curlcpp/*
+lib/rapidjson-1.0.2/*
```

Comment:
Open question: Do we need to add rapidjson if pbjson already uses it, and we can use it from pbjson lib?

I don't have a strong opinion. Just curious (: